### PR TITLE
dummy out conceptExists and xit postToCMRTasks

### PR DESCRIPTION
**Pull request by etcart**
dummies conceptExists and getOnlineResources integration test functions to eschew faulty interface with cmr, and xit out dependent integration tests
Addresses CUMULUS 3540

## PR Checklist

- [ ] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [x] Integration tests

### CODE REVIEW

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (etcart) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1]: it('pretends to work') exists to make sure the beforeAll and beforeEach are both run to the extent they can be run

File: example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js
Comment (etcart) [inline example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js:1]: as above: it('pretends to work') is here to trigger beforeAll and beforeEach

File: example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js
Comment (etcart) [inline example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js:1]: as before with it('pretends to work')

File: packages/integration-tests/cmr.js
Comment (etcart) [inline packages/integration-tests/cmr.js:145]: this returns what is expected to allow expect(false) to pass as well

File: packages/integration-tests/cmr.js
Comment (etcart) [inline packages/integration-tests/cmr.js:255]: this is attempting to reflect the rough shape of a correct return

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (jennyhliu) [reply example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1]: why not xdescribe('the PostToCmr task',

File: example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js
Comment (Jkovarik) [inline example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js:371]: I'm assuming this was proofing/diagnostic?     Looks like this should be removed.

File: example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js
Comment (Jkovarik) [inline example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js:535]: This looks like it should be removed

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (Jkovarik) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:647]: This should probably be removed

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (Jkovarik) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1099]: That's a fun consequence of stubbing this out.  The simplest edit seems fine given context, but I think we need to self-refer to this PR in the comment with this sort of thing.

```suggestion
        expect(existsInCMR).toEqual(true); // because of stubbed conceptExists  due to https://github.com/nasa/cumulus/pull/3544/  / removing CMR query from integration tests due to <ticket> 
```

File: example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js
Comment (Jkovarik) [reply example/spec/parallel/ingestGranule/ingestGranuleQueueSpec.js:535]: https://github.com/nasa/cumulus/pull/3544/files#r1447919653.      Interesting - if we commented out things such that before/after don't trigger, that implies downstream tests are relying on upstream test block behavior... which sounds like a refactor is in order, but *not* for this PR.       Please ignore the comments (resolved) that this should be removed if that's the case.

**Review by Jkovarik — COMMENTED** [review]
Looks good @etcart , just one comment fix suggestion.    Will approve later today once that's sorted if someone else doesn't get to it first.

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (etcart) [reply example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1]: there's some kinda uncomfortable interrelation in these tests. (that maybe we can revisit but is out of scope for this PR
line 610 (i'll comment it below) sets the value of postToCmrOutput in describe(' thepostToCmr task'
line 812 then uses the value of postToCmrOutput in describe('an SNS message', 

The same thing happens if the PostToCmr task has no tests actually running within it: the beforeAll never actually happens and never actually sets postToCmrOutput

as a result, we either need to restructure these a bit (which I feel is out of scope) or we need to run the beforeAll/beforeEach functions of postToCMR task, which means keeping the postToCmrTask and running one dummy test ("it pretends to work") within it

File: example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js
Comment (etcart) [reply example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js:371]: this is intentional, it's here to trigger the PostToCMrTask beforeAll() function

File: packages/integration-tests/cmr.js
Comment (jennyhliu) [inline packages/integration-tests/cmr.js:269]: Only need to fake getOnlineResources.

Comment (jennyhliu) [reply packages/integration-tests/cmr.js:269]: I suggest to keep all the old methods, and name the new ones xxxFake, and export the methods like,
getOnlineResources: getOnlineResourcesFake
It's very easy to change back when cmr is fixed.

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (jennyhliu) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:812]: remove the comment

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (jennyhliu) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:610]: remove the comment

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (npauzenga) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1092]:
```suggestion
        // because of stubbed conceptExists due to https://github.com/nasa/cumulus/pull/3544/. This should be reverted to 'false' when the stub is removed.
```

File: example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js
Comment (npauzenga) [inline example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js:1100]:
```suggestion
        // because of stubbed conceptExists due to https://github.com/nasa/cumulus/pull/3544/. This should be reverted to 'false' when the stub is removed.
```

File: packages/integration-tests/cmr.js
Comment (npauzenga) [inline packages/integration-tests/cmr.js:134]: Let's add a comment here about what's going on. Again, hopefully the person reverting these changes will know what to look for but things happen.

File: packages/integration-tests/cmr.js
Comment (npauzenga) [inline packages/integration-tests/cmr.js:303]: Same deal, just a brief documentation about what's happening might be helpful in six months when we've forgotten all of this.

**Comment by npauzenga** [comment]
Ticket to revert these changes once CMR is ready: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3555

**Review by npauzenga — APPROVED** [review]
Looks good to me. A couple open 'take it or leave it' comment suggestions but otherwise 👍

**Review by jennyhliu — APPROVED** [review]
Changelog entry?
Merge to master please.
I suggest we open a PR as blocked after this one is merged to put the tests back, so we don't forget.

**Pull request merged**
